### DISCUSSION TITLE
math: inline sum/prod is broken

### DISCUSSION THREAD

**Discussion by maple3142** [community/community#17051] [Other Feature Feedback, Questions, & Ideas] [answered]
Display Mode:

$$
f(x) = \sum_{i=0}^{n} a_i x^i
$$

$$
g(x) = \prod_{i=0}^{n} a_i x^i
$$

Inline sum: $f(x) = \sum_{i=0}^{n} a_i x^i$
Inline prod: $g(x) = \prod_{i=0}^{n} a_i x^i$
Inline integral (works): $\int_{-\infty}^{\infty} e^{-x^2}$

Screenshot:
![image](https://user-images.githubusercontent.com/9370547/169660500-e221e42d-b928-4d40-b127-9027c7a8e899.png)

**Comment by Simran-B** [comment]
For completeness, this is the input:

`$f(x) = \sum_{i=0}^{n} a_i x^i$`

`$g(x) = \prod_{i=0}^{n} a_i x^i$`

`$\int_{-\infty}^{\infty} e^{-x^2}$`

**Comment by nschloe** [comment]
I've added this to the math bug repo. https://github.com/nschloe/github-math-bugs#sumproduct-signs-in-inline-mode-or-fractions

**Comment by dpvc** [comment]
This turns out to be due to the fact that GitHub is removing some attributes from the MathJax output.  In this case, the `limits="false"` attribute has been removed from the HTML nodes that enclose the sum and product characters, so the needed CSS to style those character properly is not being applied.

**Comment by rogerwwww** [comment, marked as answer]
Seems that this issue has been fixed
